File: C#/Card.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace C_
{
    internal class Card
    {
        private static readonly string[] Suits = { "heart", "diamond", "club", "spade" };
        private static readonly string[] Colors = { "red", "black" };
        private static Random random = new Random();
        private int number;
        private string color;
        private string suit;

        public Card()
        {
            this.number = this.GenerateRandomNumber(1, 10);//Get a random number
            this.color = Colors[this.GenerateRandomNumber(0, 2)];//Get a random color from the list
            this.suit = Suits[this.GenerateRandomNumber(0, 4)];//Get a random suit from the list
        }

        //Number lottery function
        private int GenerateRandomNumber(int min, int max)
        {
            return random.Next(min, max);
        }

        //Describe the object
        public override string ToString()
        {
            return $"Card{{ number={number}, color={color}, suit={suit} }}";
        }

        //Draw a card
        public void DrawCard()
        {
            Console.WriteLine("__________\n  " + this.suit + "\n  " + this.number + "\n  " + this.color + "\n__________");
        }

        //Getters 
        public int GetNumber()
        {
            return number;
        }

        public string GetColor()
        {
            return color;
        }

        public string GetSuit()
        {
            return suit;
        }

    }
}

File: Node.js/card.js
class Card {
    static SUITS = ['heart', 'diamond', 'club', 'spade'];
    static COLORS = ['red', 'black'];
    constructor() {
        this.number = this.generateRandomNumber(0, 9);//Get a random number
        this.suit = Card.SUITS[this.generateRandomNumber(0, 3)];//Get a random suit from the list
        this.color = Card.COLORS[this.generateRandomNumber(0, 1)];//Get a random color from the list
    }
    //Number lottery function
    generateRandomNumber(min, max) {
        return Math.floor(Math.random() * (max - min + 1)) + min;
    }

    //Describe the object
    toString() {
        return `Card [number:${this.number}, suit:${this.suit}, color:${this.color}]`;
    }
    //Draw a card
    drawCard() {
        console.log(`______\n   ${this.suit}\n   ${this.number}\n   ${this.color}   \n______`);
    }
}
export { Card }
